packages/trip-details/src/styled.js

```diff
@@ -44,7 +44,15 @@ export const TNCFareCompanies = styled.span`
 
 export const Timing = styled.span``;
 
-export const TransitFare = styled.span``;
+export const TransitFare = styled.details`
+  display: inline-block;
+
+  > span {
+    display: block;
+    padding-left: 2ch;
```

Comment:
Some of the CSS changes cause a slight, barely visible mislalignment:

![image](https://user-images.githubusercontent.com/56846598/140088276-2ca65f72-2bd7-408f-890b-54adc8e908c1.png)

Reply:
Good catch thanks! Looks like there's a `0.25ch` padding on the caret from somewhere else.

Reply:
Fixed in 088cc5ff030b873cf8e3b93a64a5a84377ce7142